sapi/fpm/tests/status-inherited-settings.phpt

```diff
@@ -0,0 +1,75 @@
+--TEST--
+FPM: Status pool is inheriting FPM setting overrides test
+--SKIPIF--
+<?php include "skipif.inc"; ?>
+--INI--
+expose_php=Off
+--FILE--
+<?php
+
+require_once "tester.inc";
+
+$cfg = <<<EOT
+[global]
+error_log = {{FILE:LOG}}
+[unconfined]
+listen = {{ADDR}}
+pm = static
+pm.max_children = 1
+pm.status_listen = {{ADDR[status]}}
+pm.status_path = /status
+php_flag[expose_php] = Off
+EOT;
+
+$expectedStatusData = [
+    'pool'                 => 'unconfined',
+    'process manager'      => 'static',
+    'listen queue'         => 0,
+    'max listen queue'     => 0,
+    'idle processes'       => 1,
+    'active processes'     => 0,
+    'total processes'      => 1,
+    'max active processes' => 1,
+    'max children reached' => 0,
+    'slow requests'        => 0,
+];
+
+$tester = new FPM\Tester($cfg);
+$tester->start();
+$tester->expectLogStartNotices();
+$response = $tester->request('', [], '/status', '{{ADDR[status]}}');
+$response->expectNoHeader('X-Powered-By');
+$tester->terminate();
+$tester->expectLogTerminatingNotices();
+$tester->close();
+
+$cfg = <<<EOT
+[global]
+error_log = {{FILE:LOG}}
+[unconfined]
+listen = {{ADDR}}
+pm = static
+pm.max_children = 1
+pm.status_listen = {{ADDR[status]}}
+pm.status_path = /status
+php_flag[expose_php] = On
+EOT;
+
+$tester = new FPM\Tester($cfg);
+$tester->start();
+$tester->expectLogStartNotices();
+$response = $tester->request('', [], '/status', '{{ADDR[status]}}');
+$response->expectHeader('X-Powered-By', '|^PHP/8|', true);
+$tester->terminate();
+$tester->expectLogTerminatingNotices();
+$tester->close();
```

Comment:
might be better to do 2 tests for this instead:

- status-listen-expose-php-on.phpt
- status-listen-expose-php-off.phpt

Reply:
Good idea - now done.